api/src/services/opportunities_v1/get_opportunity.py

```diff
@@ -50,7 +56,14 @@ def get_opportunity(db_session: db.Session, opportunity_id: int) -> Opportunity:
         db_session, opportunity_id, load_all_opportunity_summaries=False
     )
 
-    pre_sign_opportunity_file_location(opportunity.opportunity_attachments)
+    attachment_config = AttachmentConfig()
+    if attachment_config.cdn_url is not None:
+        for opp_att in opportunity.opportunity_attachments:
+            opp_att.download_path = convert_public_s3_to_cdn_url(  # type: ignore
+                opp_att.file_location, attachment_config.cdn_url
+            )
+    else:
+        pre_sign_opportunity_file_location(opportunity.opportunity_attachments)
```

Comment:
I'm not sure there will be a case where it's not set once we merge this. (NOTE - we should first verify with @coilysiren that the CDN is fully working before we merge this - I think it's still in progress)